lib.ts

```diff
@@ -15,4 +15,16 @@ export const newAnchorage = (width: number, height: number): Anchorage => {
   }
 
   return container;
+}
+
+export const findPos = (container: Anchorage): Position | null  => {
+  for (let i = 0; i < container.length; i++) {
```

Comment:
since anchorage has a known size, why not represent it as an object with Dimensions?

Reply:
I'm not sure how I keep track of remaining space in this anchorage then) I felt that I can replace zeroes with any other values to do that.
Like your proposal is to make let's say 3x3 anchorage look not like this:

**const anchorage = [
  [0, 0, 0],
  [0, 0, 0],
  [0, 0, 0],
]**

But rather like this:

**const anchorage = { x: 3, y: 3}** 

Right?